app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) < 3 or command_list[0] != "mv":
+        print("command wrong")
+    if "/" not in command_list[2]:
+        os.rename(command_list[1], command_list[2])
+
+    patch, name = os.path.split(command_list[2])
+    os.rename(command[1], name)
+    os.makedirs(patch)
+    os.replace(name, command_list[2])
```

Comment:
If the command list length will be less then 3 -> we will only print something and continue. After that on the line 11 we will get failed. Add some returns where it's needed.